bagit.xml

```diff
@@ -1,819 +1,800 @@
-bag checksum algorithm.</t>
-<t>
-Each line of a payload manifest file &must; be of the form:
-</t>
-
-<figure>
- <artwork>
-CHECKSUM FILENAME
- </artwork>
-</figure>
-
-<t>
-where FILENAME is the pathname of a file relative to the base directory
-and CHECKSUM is a hex-encoded checksum calculated according to <spanx
-style="emph">algorithm</spanx> over every octet in the file. The hex-encoded
-checksum &may; use uppercase and/or lowercase letters.  The slash
-character ('/') &must; be used as a path separator in FILENAME. One
-or more linear whitespace characters (spaces or tabs) &must; separate
-CHECKSUM from FILENAME.  An asterisk ('*') &may; preceed FILENAME for
-interoperability on some platforms (see <xref target="sec-checksum-tools"
-/>). There is no limitation on the length of a pathname. The payload
-manifest &must-not; reference files outside the payload directory. If
-a FILENAME includes a newline (LF), a carriage return (CR), or carriage
-return plus newline (CRLF) it &must; be percent-encoded 
-<xref target="RFC3986"/>.
-
-</t>
-
-<t>
-Payload manifests only include the pathnames of files.  Because of this,
-a payload manifest cannot reference empty directories.  To account for 
-an empty directory, a bag creator may wish to include at least one file 
-in that directory; it suffices, for example, to include a zero-length 
-file named ".keep".
-</t>
-</section> <!-- /Payload Manifest -->
-</section> <!-- /Required Elements -->
-
-<section title="Optional Elements" anchor="sec-optional-elements">
-<section title="Tag Manifest: tagmanifest-&lt;alg&gt;.txt">
-<!-- WARNING: This section should be kept in relative sync with the
-              section on Payload Manifests.
--->
-<t>
-A tag manifest is a tag file that lists other tag files and checksums for
-those tag files generated using a particular bag checksum algorithm.
-A bag &may; contain one or more tag manifests.
-A tag manifest file &must; have a name of the form
-"tagmanifest-<spanx style="emph">algorithm</spanx>.txt", where
-<spanx style="emph">algorithm</spanx> is a string specifying
-the bag checksum algorithm used in that manifest, such as:
-</t>
-
-<figure>
- <artwork>
+  </artwork>
+          </figure>
+          <t>
+            Each line of a payload manifest file &must; be of the form:
+          </t>
+          <figure>
+            <artwork>checksum filename</artwork>
+            <postamble>
+    where <spanx style="emph">filename</spanx> is the pathname of a file
+    relative to the base directory, and <spanx style="emph">checksum</spanx> is
+    a hex-encoded checksum calculated according to
+    <spanx style="emph">algorithm</spanx> over every octet in the file.
+  </postamble>
+          </figure>
+          <t>
+            <list style="symbols">
+              <t>The hex-encoded checksum &may; use uppercase and/or lowercase letters.</t>
+              <t>The slash character ('/') &must; be used as a path separator
```

Comment:
@acdha I believe that to be the case. For the java implementation I believe we will collapse down the filepath so that any `./` or `../` will be removed so moving to a new version should be fairly simple. But I agree this should probably wait until 2.0 since we will be doing breaking changes in that version anyway